MWStoryGitFile.txt

```diff
@@ -19880,7 +19880,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sun Nov 8 16:33:20 GMT+0100 2020_12231"MainText""
 "Sun Nov 8 17:05:01 GMT+0100 2020_12009"MainText"And now for a very special feature, something that Malia loves."
 "Sun Nov 8 17:05:01 GMT+0100 2020_12010"MainText"I love it too!"
-"Sun Nov 8 17:05:01 GMT+0100 2020_12011"MainText"You know what I am talking about?"
+"Sun Nov 8 17:05:01 GMT+0100 2020_12011"MainText"You know what I am talkingabout?"
```

Comment:
You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<